js/src/tooltip.js

```diff
@@ -301,7 +286,7 @@ class Tooltip extends BaseComponent {
       }
 
       this._element.removeAttribute('aria-describedby')
-      EventHandler.trigger(this._element, this.constructor.Event.HIDDEN)
+      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
```

Comment:
I don't know, but can't this be `this.eventName` instead of `this.constructor.eventName` everywhere?

Reply:
We need to call it statically so we can use it outside the class (at least to what I have seen till now)